EpiserverRedirects/UrlRewritePlugin/Menu/UrlRedirectsMenuController.cs

```diff
@@ -14,12 +14,12 @@ public UrlRedirectsMenuController(IBootstrapper bootstrapper)
             _bootstrapper = bootstrapper;
         }
 
-        [MenuItem(MenuPaths.Global + "/urlRedirects",
+        [MenuItem(MenuPaths.Global + "/episerverRedirects",
            SortIndex = SortIndex.Last + 20,
-           Text = "Url Redirects")]
+           Text = "Episerver Redirects")]
```

Comment:
Don't think that inside of Episerver menu we need to state that we are inside of Episerver ;)
@mgrzyb what do you think?